python/orca/test/bigdl/orca/learn/ray/experimental/test_mmcv_estimator_ray_backend.py

```diff
@@ -218,6 +219,42 @@ def test_run_with_batch_processor(self):
         print(f"dLoss: {dloss}")
         assert dloss < 0
 
+    def test_03_save_ckpt(self):
+        model = Model()
+        optimizer = optim.SGD(model.parameters(), lr=0.001, momentum=0.9)
+        cfg = dict(
+            model=model,
+            optimizer=optimizer,
+            batch_processor=None,
+            lr_config=dict(policy='step', step=[2, 3]),
+            optimizer_config=dict(grad_clip=None),
+            checkpoint_config=dict(interval=1),
+            log_config=dict(interval=4, hooks=[dict(type='TextLoggerHook')])
+        )
+        estimator = get_estimator(runner_creator, cfg)
+        estimator.run([train_dataloader_creator], [('train', 1)])
+        assert os.path.exists(os.path.join(TEMP_WORK_DIR, "epoch_1.pth"))
+        assert os.path.exists(os.path.join(TEMP_WORK_DIR, "epoch_2.pth"))
+        assert os.path.exists(os.path.join(TEMP_WORK_DIR, "epoch_3.pth"))
+        assert os.path.exists(os.path.join(TEMP_WORK_DIR, "epoch_4.pth"))
+
+    def test_04_load_ckpt(self):
+        model = Model()
+        optimizer = optim.SGD(model.parameters(), lr=0.001, momentum=0.9)
+        cfg = dict(
+            model=model,
+            optimizer=optimizer,
+            batch_processor=None,
+            lr_config=dict(policy='step', step=[2, 3]),
+            optimizer_config=dict(grad_clip=None),
+            checkpoint_config=None,
+            log_config=dict(interval=4, hooks=[dict(type='TextLoggerHook')])
+        )
+        estimator = get_estimator(runner_creator, cfg)
+        estimator.load_checkpoint(os.path.join(TEMP_WORK_DIR, "epoch_4.pth"))
```

Comment:
this may be existing at start, for pytest.fixture tears down when whole package is finished (scope='package)?